django/core/management/commands/runserver.py

```diff
@@ -183,18 +183,23 @@ def on_bind(self, server_port):
         version = self.get_version()
         print(
             f"{now}\n"
-            f"Django version {version}, using settings {settings.SETTINGS_MODULE!r}\n"
-            f"Starting development server at {self.protocol}://{addr}:{server_port}/\n"
+            f"Django version {version}, using settings"
+            f"{settings.SETTINGS_MODULE!r}\n"
+            f"Starting WSGI development server at "
+            f"{self.protocol}://{addr}:{server_port}/\n"
```

Comment:
These need to span the full allowable length
```suggestion
            f"Django version {version}, using settings {settings.SETTINGS_MODULE!r}\n"
            f"Starting WSGI development server at "
            f"{self.protocol}://{addr}:{server_port}/\n"
```

Reply:
@cliffordgama Converting to draft while I address CI failures and review feedback. I’ll update this shortly.